Jenkinsfile

```diff
@@ -0,0 +1,88 @@
+def workpath_linux = "/src/github.com/deis/workflow-cli"
+def keyfile = "tmp/key.json"
+
+def upload_artifacts = { String filepath ->
+	withCredentials([[$class: 'FileBinding', credentialsId: 'e80fd033-dd76-4d96-be79-6c272726fb82', variable: 'GCSKEY']]) {
+		sh "cat \"\${GCSKEY}\" > ${filepath}"
+		sh "make upload-gcs"
+	}
+}
+
+node('windows') {
+	def gopath = pwd() + "\\gopath"
+	env.GOPATH = gopath
+	def workdir = gopath + "\\src\\github.com\\deis\\workflow-cli"
+
+	def pscmd = { String cmd ->
+		"powershell -NoProfile -ExecutionPolicy Bypass -Command \"${cmd}\""
+	}
+
+	dir(workdir) {
+		stage 'Checkout Windows'
+			checkout scm
+		stage 'Install Windows'
+			bat pscmd('.\\make bootstrap')
+		stage 'Test Windows'
+			bat pscmd('.\\make test')
+	}
+}
+
+node('linux') {
+	def gopath = pwd() + "/gopath"
```

Comment:
minor nit: what do you think about wrapping this linux gopath setup into its own closure, as it appears used identically in 3 spots?